pkg/stackit_api_manager/client/client.go

```diff
@@ -346,9 +352,21 @@ func (c *APIClient) prepareRequest(
 
 		// AccessToken Authentication
 		if auth, ok := ctx.Value(ContextAccessToken).(string); ok {
-			localVarRequest.Header.Add("Authorization", "Bearer "+auth)
-		}
+			if auth == "" {
```

Comment:
Just for readability: 
```suggestion
			switch string.Split(auth, " "){ 
			// comment explaining 
			case 1: prefix Bearer
			// comment explaining 
			case 2: validate auth[0] == Bearer
			default: error
			
```